pkg/storaged/utils.js

```diff
@@ -1031,3 +1057,31 @@ export function is_mounted_synch(block) {
 export function for_each_async(arr, func) {
     return arr.reduce((promise, elt) => promise.then(() => func(elt)), Promise.resolve());
 }
+
+/*
+ * Get mount points for a given org.freedesktop.UDisks2.Filesystem object
+ *
+ * This generalises getting the given MountPoints of a Filesystem for btrfs and
+ * other filesystems, for btrfs we want to know if a subvolume is mounted
+ * anywhere. UDisks is currently not aware of subvolumes and it's Filesystem
+ * object gives us the MountPoints for all subvolumes while we want it per
+ * subvolume.
+ *
+ * @param {Object} block_fsys
+ * @param {Object|null} subvol
+ * @returns {Array} an array of MountPoints
```

Comment:
array of strings, no?

Reply:
Yeah, and a MountPoint is a string, but you usually describe what kind of data you return.

Seems this is not possible to describe yet.. https://github.com/jsdoc/jsdoc/issues/1073